packages/hierarchy-browser/package.json

```diff
@@ -22,14 +22,17 @@
 		"not op_mini all"
 	],
 	"dependencies": {
+		"@essex-js-toolkit/hooks": "workspace:packages/hooks",
 		"@thematic/core": "^0.8.0",
 		"@thematic/fluent": "^0.8.0",
 		"@thematic/react": "^0.8.0",
 		"@types/d3-array": "^2.2.0",
```

Comment:
`@types/<x>` dependencies should go into devDeps and peerDeps

Reply:
I updated the yarn version as followed (but happy to make changes if this doesn't work):
releases:
  "@essex-js-toolkit/hierarchy-browser": major

declined:
  -"@essex-js-toolkit/dev"